### PR TITLE
feat(deel): Adds Deel integration onboarding guide

### PR DESCRIPTION
**Pull request by sshariff01**
### What

- Adds an onboarding guide for the Deel integration
- This doc will be accessible only via direct link for the time being

### Why

Onboarding guide for our Sales and Success teams to share with customers who
are interested in setting up an integration with Deel via Finch.

### CODE REVIEW

File: docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+> Integrating Deel via Finch makes your organization eligible for the Deel Partner Program. This includes go-to-market motions and revenue share opportunities with Deel. If you are interested or have already built an app, reach out here to learn more about the program.
+
+## 1. Create a Deel account
+
+If your company does not already have an account with Deel, you’ll need to create one. To create an account, follow the instructions below.
+
+1. Navigate to https://app.deel.com/signup in your browser
+1. Select the **I’m A Business** option and click **Next**
+
+    ![deel_signup_1.png](../../../../assets/images/deel_signup_1.png)
```
Comment (alexromano) [inline docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md:26]: you can use
```
<!--
focus: false
-->
```
if you want to get rid of the border/focus functionality on the images

Comment (alexromano) [reply docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md:26]: e.g.
```
<!--
focus: false
-->
![](../../assets/images/howFinchWorks4.png)
```

File: docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Integrating with Deel through Finch is a simple 3-step process! To get set up, you need to:
+
+1. Create a Deel account
+1. Create a Deel OAuth application
+1. Connect your Deel OAuth application to Finch
+
+This page explains each of these steps in detail. Happy integrating!
+
+> Integrating Deel via Finch makes your organization eligible for the Deel Partner Program. This includes go-to-market motions and revenue share opportunities with Deel. If you are interested or have already built an app, reach out here to learn more about the program.
```
Comment (alexromano) [inline docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md:17]: missing link for "here"?

File: docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+1. Either **Sign Up Using Google SSO** or fill in the form below and select **Create Your Deel Account**
+
+    ![deel_signup_2.png](../../../../assets/images/deel_signup_2.png)
+
+1. If your form was submitted successfully, you’ll be taken to a confirmation page. You should now have received an email confirmation in your inbox. Open this email and follow the ensuing instructions to log into your generated account. 
+
+    ![deel_signup_3.png](../../../../assets/images/deel_signup_3.png)
+
+1. After logging into your account, follow the ensuing instructions to complete your account setup. When prompted with the solutions selection page, you may select ********Skip,******** and proceed with the remaining steps to complete your setup.
```
Comment (alexromano) [inline docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md:37]: do you need the extra **********?

**Review by alexromano — APPROVED** [review]

File: docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md (hunk 3 above)
Comment (sshariff01) [reply docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md]: Nope that was copy-pasta from Notion. Updated.

**Pull request merged**